docs/tutorial/ecommerce-tutorial/index.md

```diff
@@ -171,6 +171,10 @@ const Checkout = class extends React.Component {
 export default Checkout
 ```
 
+Be sure to enable the Checkout client-only integration on your Stripe dashboard to use the redirectToCheckout integration, you may do that in this url: https://dashboard.stripe.com/account/checkout/settings
```

Comment:
```suggestion
Be sure to enable the "Checkout client-only integration" on your Stripe dashboard to be able to use the redirectToCheckout integration. You can activate the integration in your [Stripe settings](https://dashboard.stripe.com/account/checkout/settings).
```

Let's use a nicer link to the settings and clarify the wording a bit.

Reply:
Thank you @LekoArts I just pushed the requested changes.